docs/project_defn.rst

```diff
@@ -298,29 +299,28 @@ using `GCP Batch <https://cloud.google.com/batch>`_ and `Cloud Run <https://clou
        repository.
 *  ``job_environment``: Optional. Specifies the computing requirements for each simulation.
 
-    *  ``vcpus``: Number of CPUs to allocate for running each simulation. Default: 1.
-    *  ``memory_mib``: Amount of RAM memory needed for each simulation in MiB. Default: 1024.
-       For large multifamily buildings this works better if set to 2048.
+    *  ``vcpus``: Optional. Number of CPUs to allocate for running each simulation. Default: 1.
+    *  ``memory_mib``: Optional. Amount of RAM memory to allocate for each simulation in MiB.
+       For large multifamily buildings this works better if set to 2048. Default: 1024.
```

Comment:
Suggest removing the 2048 because almost everything has worked fine with 1024 in resstock.